tools/hardware/killerbee_msfrelay.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```

Comment:
there is no python3 release of killerbee, making this conversion not really possible at this time.